### DISCUSSION TITLE
Get texture from Text

### DISCUSSION THREAD

**Discussion by tobyshooters** [pixijs/pixijs#10382] [Q&A]
With Pixi v6, we used to use the following pattern to create a sprite from a Text object:
```
const text = new Text(...);
const texture = new Sprite(text.texture);
```

This is some old code, I don't remember exactly the decision here, but I recall this being an optimization to avoid re-renders.
Is it possible to get the `texture` from the Text object still? Or should I change the pattern?

**Comment by IcedDog** [comment]
In 7.0, `Text.texture` still exist.
In 8.0 however, though the texture can be obtained through its view container, it probably isn't the texture the text use.
```
const text = new Text(...);
const texture = renderer.generateTexture(text);
```
